### PR TITLE
Translate Spanish FAQ and Tutorial

### PR DESCRIPTION
**Pull request by AlfonsoSB**
@Ken-Lo this is the last part to merge.
I have included both the FAQ for merchants and the French Tutorial in Spanish, to help people in Self-serve and to have more content for people to search for us.

### CODE REVIEW

File: app/pages/faq/pro-dashboard/mandate-creation/faq-pro-dashboard-mandate-creation.es.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          electrónico seguro
+          </li>
+          <li>
+          Indicando directamente los datos de tus clientes, recopilados anteriormente a través de un
+          mandato por escrito o por teléfono (reservado a los clientes de GoCardless Pro y Enterprise)
+          </li>
+        </ul>
+
+        <p className='para'>
+          APara comenzar, ve a la pestaña Cliente en el panel de control y haz clic en
```
Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/mandate-creation/faq-pro-dashboard-mandate-creation.es.js]: APara or Para?

Comment (Ken-Lo) [inline app/pages/faq/pro-dashboard/mandate-creation/faq-pro-dashboard-mandate-creation.es.js]: I'm changing this to Para as I think it's a typo... let me know if I'm wrong!

Comment (AlfonsoSB) [inline app/pages/faq/pro-dashboard/mandate-creation/faq-pro-dashboard-mandate-creation.es.js]: Yes, it's a Typo. Thanks!

File: app/messages/es.js [hunk 2]
```diff
@@ -106,6 +106,123 @@ export default {
     description: 'GoCardless es la manera más sencilla de pagar online. Realiza tus cobros directamente desde la cuenta bancaria de tus clientes. Ahorra tiempo y mejora tu flujo de caja.',
     explainer: 'Simplificamos las Domiciliaciones Bancarias',
   },
+  faq: {
+    title: 'FAQ - Preguntas más comunes',
+    sidebar: '¿No encuentras respuesta? Llama a nuestro equipo de soporte al',
+  },
+  faq_merchants: {
```
Comment (Ken-Lo) [inline app/messages/es.js]: @AlfonsoSB Could you kindly translate the messages in `app/messages/es.js` between lines 113 and 225?

File: app/router/routes.js [hunk 3]
```diff
@@ -1016,6 +1016,9 @@ export const config = Immutable.fromJS([
       de: {
         path: '/faq/haendler',
       },
+      es: {
```
Comment (Ken-Lo) [inline app/router/routes.js]: @AlfonsoSB You will also need to go through `app/router/routes.js` and translate all of the new entries I've added (at the moment, the URL still shows in English)

**Comment by Ken-Lo** [comment]
Nice one @AlfonsoSB! I've made a bunch of changes from my side - just three things remaining before we can merge:

- [ ] Could you kindly translate the messages in `app/messages/es.js` between lines 113 and 225?
- [ ] You will also need to go through `app/router/routes.js` and translate all of the new items I've added
- [ ] Once you've done this, just have another read through before merging to dev!

**Comment by AlfonsoSB** [comment]
@Ken-Lo In es.js there is
  faq_pro_dashboard_paylinks_tracking: {
    title: 'FAQ - Paylinks creation',
    nav_title: 'Paylinks creation',
    description: '',

However, I don't see any page called Paylinks tracking.

**Comment by Ken-Lo** [comment]
@AlfonsoSB Ah sorry I'm not sure why that is there - you can remove the `faq_pro_dashboard_paylinks_tracking` section of that file!

**Comment by AlfonsoSB** [comment]
@Ken-Lo Done! I believe everything is verified now. Will check again once it is published

**Pull request merged**